packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,51 @@
+When not working with an array as a source of data, an array of templates to
+render can be manually built, for example, by using a `for ... of` loop.
+
+{% switchable-sample %}
+
+```ts
+// my-element.ts
+const templates = [];
+for (const item of this.items) {
+  templates.push(html`<li>${item}</li>`);
+}
+```
+
+```js
+// my-element.js
+const templates = [];
+for (const item of this.items) {
+  templates.push(html`<li>${item}</li>`);
+}
+```
+
+{% endswitchable-sample %}
+
+It can then be placed inside the template returned from the render method.
```

Comment:
This might be too advanced but there's an opportunity here to mention (perhaps in an aside) that Lit templates can host any javascript expression, but if you're using statements like this, you can build up a result to later place in a template (statements can also be easily used by returning a result from a function... hopefully there's a link in the docs we could just call out to?).

Reply:
Added an aside stating any javascript expressions resulting in valid values for expressions with link to doc.